src/test/java/com/battleship/backend/BattleshipApplicationTests.java

```diff
@@ -15,19 +16,21 @@
 	@Autowired
 	private WelcomeController welcomeController;
 
+	@Autowired
+	private BoardController boardController;
+
 	@Test
 	void contextLoads() {
 	}
 
 	@Test
-	void testTest(){
-		int five = 5;
-		assertEquals(8, five + 3);
+	void testWelcomeControllerExists() throws Exception {
+		assertThat(welcomeController).isNotNull();
 	}
 
 	@Test
-	void testWelcomeControllerExists() throws Exception {
-		assertThat(welcomeController).isNotNull();
+	void testBoardControllerExists() throws Exception {
+		assertThat(boardController).isNotNull();
 	}
```

Comment:
I'm not following the purpose of these tests.  Why do we have these tests?  If you have the relevant controller tests, why do you need these tests?